02_RegressionClassification/RegressionClassification.ipynb

```diff
@@ -0,0 +1,1734 @@
+{
```

Comment:
<p>Recap slides - Example of each of the concepts - as a summary - interactive. (Ask the difference between the different examples and how they know it's each concept.) Visualize for ""imaginary"" problems. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BNL-Fermi-Summer-School/tutorials_2024/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>